README.md

```diff
@@ -83,6 +83,136 @@ By default, the `tempesta_fw.conf` from this directory is used.
 See `tempesta_fw.conf` for the list of available options and their descriptions.
 
 
+### Listening address
+
+Tempesta listens to incoming connections on specified address and port. The syntax is as follows:
+```
+listen <PORT> | <IPADDR>[:PORT]
+```
+`IPADDR` may be either IPv4 or IPv6 address. Host names are not allowed. IPv6 address must be enclosed in square brackets (e.g. "[::0]" but not "::0"). If only `PORT` is specified, then address 0.0.0.0 (but not [::1]) is used.
+If only `IPADDR` is specified, then default HTTP port 80 is used.
+
+Tempesta opens one socket for each `listen` directive. Multiple `listen` directives may be defined to listen on multiple addresses/ports. If `listen` directive is not defined in the configuration file, then by default Tempesta listens on IPv4 address 0.0.0.0 and port 80, which is an equivalent to `listen 80` directive.
+
+Below are examples of `listen` directive:
+```
+listen 80;
+listen [::0]:80;
+listen 127.0.0.1:8001;
+listen [::1]:8001;
+```
+
+### Servers
+
```

Comment:
Would be good to make this and all the new chapters below sub-chapters of one common chapter (also 3rd level) like "Server Load Balancing"